### ISSUE TITLE
malware target_ref not found

### ISSUE THREAD

**Issue by Radu3000** [mitre/cti#92]
Consider the following relationship:

```
        {
            "type": "relationship",
            "target_ref": "malware--c41a8b7c-3e42-4eee-b87d-ad8a100ee878",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "created": "2019-07-10T15:47:19.659Z",
            "id": "relationship--53364899-1ea5-47fa-afde-c210aed64120",
            "source_ref": "intrusion-set--8a831aaa-f3e0-47a3-bed8-a9ced744dd12",
            "modified": "2019-07-16T15:35:21.086Z",
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "external_references": [
                {
                    "source_name": "Lookout Dark Caracal Jan 2018",
                    "description": "Blaich, A., et al. (2018, January 18). Dark Caracal: Cyber-espionage at a Global Scale. Retrieved April 11, 2018.",
                    "url": "https://info.lookout.com/rs/051-ESQ-475/images/Lookout_Dark-Caracal_srr_20180118_us_v.1.0.pdf"
                }
            ],
            "relationship_type": "uses"
        }
```

However there is no "id": "malware--c41a8b7c-3e42-4eee-b87d-ad8a100ee878" in the https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json

- Any idea why?

Thanks,
Radu

**Comment by isaisabel** [comment]
Hi @Radu3000, 

Please see #74, which covers this exact issue.

**Comment by isaisabel** [comment]
Hi @Radu3000,

This has been fixed in ATT&CK v7.1. There were a few other occurrences of this bug which are noted in [the update log](https://github.com/mitre-attack/attack-website/blob/7fd4c84af4be99657cfbc7b7b14b5fa73092e681/CHANGELOG.md#attck-content-version-71).

**Comment by Radu3000** [comment]
Thank you 
I see no more malware--c41a8b7c-3e42-4eee-b87d-ad8a100ee878 in the enterprise json - but I see it in mobile json.
